cmd/world/root/root_test.go

```diff
@@ -99,14 +100,12 @@ func TestExecuteDoctorCommand(t *testing.T) {
 	}
 }
 
-func TestCreateStartStopPurge(t *testing.T) {
+func TestCreateStartStopRestartPurge(t *testing.T) {
 	// Create Cardinal
 	gameDir, err := os.MkdirTemp("", "game-template")
+	os.Chdir(gameDir)
```

Comment:
The Chdir command should happen after the assert. If MkdirTemp does return an error, gameDir is likely invalid.

Reply:
updated